great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -523,31 +559,37 @@ def add_column_row_condition(
         self, domain_kwargs, column_name=None, filter_null=True, filter_nan=False
     ):
         # We explicitly handle filter_nan & filter_null for spark using a spark-native condition
-        if "row_condition" in domain_kwargs and domain_kwargs["row_condition"]:
-            raise GreatExpectationsError(
-                "ExecutionEngine does not support updating existing row_conditions."
-            )
 
         new_domain_kwargs = copy.deepcopy(domain_kwargs)
         assert "column" in domain_kwargs or column_name is not None
         if column_name is not None:
             column = column_name
         else:
             column = domain_kwargs["column"]
-        if filter_null and filter_nan:
-            new_domain_kwargs[
-                "row_condition"
-            ] = f"NOT isnan({column}) AND {column} IS NOT NULL"
-        elif filter_null:
-            new_domain_kwargs["row_condition"] = f"{column} IS NOT NULL"
-        elif filter_nan:
-            new_domain_kwargs["row_condition"] = f"NOT isnan({column})"
-        else:
+
```

Comment:
@anthonyburdi If possible, we should try to use the `SQLAlchemy` way.  Happy to discuss.  Thanks.